### PR TITLE
DSET-4485: Adding configs to DataSet Grafana Cloud returns TypeError 

### PR DESCRIPTION
**Pull request by jmakar-scalyr**

### CODE REVIEW

File: src/ConfigEditor.tsx [hunk 1]
```diff
@@ -76,7 +75,7 @@ export class ConfigEditor extends PureComponent<Props, State> {
               labelWidth={8}
               inputWidth={20}
               onChange={this.onURLChange}
-              value={jsonData.scalyrUrl}
+              value={jsonData.scalyrUrl || 'https://app.scalyr.com/'}
```
Comment (tomaz-s1) [inline src/ConfigEditor.tsx:78]: LGTM, this was also my first suspicion when I quickly glanced over the git repo history and checking code which touched ``scalyrUrl`` related stuff - https://github.com/scalyr/scalyr-grafana-datasource-plugin/commit/4c548155e952cf75ce3fa541f00171fd3d7748e0#diff-1d0cc51dac0e69e79828e482663c8760447400c4cba9e4820ffb7f9f933312aeL78

**Review by tomaz-s1 — DISMISSED** [review]

File: src/ConfigEditor.tsx (hunk 1 above)
Comment (tomaz-s1) [reply src/ConfigEditor.tsx:78]: Once the fix is confirmed and we get a chance, it would be great if we could add a test case for it.

Comment (tomaz-s1) [inline src/ConfigEditor.tsx:78]: (minor small possible future improvement) We could define this value as part of a constant and reference it everywhere.

Comment (jmakar-scalyr) [reply src/ConfigEditor.tsx:78]: Agree, will defer this.  Prefer all the frontend code gets a review (will make a ticket for this as well)

**Pull request merged**